draft-ietf-core-oscore-edhoc.md

```diff
@@ -280,6 +284,8 @@ The client prepares an EDHOC + OSCORE request as follows.
 
 With the same server, the client SHOULD NOT have multiple simultaneous outstanding interactions (see {{Section 4.7 of RFC7252}}) such that: they consist of an EDHOC + OSCORE request; and their EDHOC data pertain to the EDHOC session with the same connection identifier C_R.
 
+An exception applies for clients that operate under particular time constraints over particularly unreliable networks, thus raising the chances to promptly complete the EDHOC execution with the server through multiple, simultaneous EDHOC + OSCORE requests. As discussed in {{security-considerations}}, this does not have any impact in terms of security.
```

Comment:
This seems like this procedure is a normative part of EDHOC-OSCORE.  Maybe parenthesize the whole paragraph and s/applies/might apply/...  Just to tone it down a bit.

Reply:
Done.